cosmovisor/args.go

```diff
@@ -146,6 +148,9 @@ func GetConfigFromEnv() (*Config, error) {
 	if err := cfg.validate(); err != nil {
 		return nil, err
 	}
+
+	cfg.PreupgradeMaxRetries, _ = strconv.Atoi(os.Getenv(envPreupgradeMaxRetries))
+
```

Comment:
nit: I'd probably return the error here if non-nil.

Reply:
then cosmovisor would error out on `PREUPGRADE_MAX_RETRIES` not being set. But not all upgrades would need preupgrade. 

Should a not-needed setting prevent cosmovisor from running?

Reply:
Check if its not empty first.

Reply:
👍🏻